frontend/src/components/CreateAccountForm.vue

```diff
@@ -7,18 +7,22 @@ import { Form, FormField, type FormSubmitEvent } from '@primevue/forms';
 import { InputText, Message, Button } from 'primevue';
 import { useToast } from 'primevue/usetoast';
 import Toast from 'primevue/toast';
+import {hashPassword} from "@/components/password";
 const toast = useToast();
 const onFormSubmit = (event: FormSubmitEvent) => {
     let username = event.states.username.value;
     let email = event.states.email.value;
     let password = event.states.password.value;
+    if (!username || !email || !password) {
+      console.log("Empty fields");
+      toast.add({severity: 'error', summary: 'Credentials cannot be empty', life: 3000});
+      return;
+    }
```

Comment:
I would add form validation in addition to this toast message, see `ValidateOn` from PrimeVue library: https://primevue.org/forms/#validateon

Reply:
Created an issue of this as I have big problems with making it work for now.